### ISSUE TITLE
Overall reduced mIoU value

### ISSUE THREAD

**Issue by cskhy16** [cailile/Revisiting-Superpixels-for-Active-Learning#1]
I am interested in the field of active learning with superpixel and try to reproduce the result of cityscapes dataset described in the result.png. Thanks to your kind explanation, I could easily run the code, but I obtain reduced mIoU values overall.

result.png seems like:
  1. Sp+Do+Random - 59% (100k), 65.5% (200k)
  2. Rec+Pr+Random - 53% (100k), 61% (200k)

When I run the code:
  1. Sp+Do+Random - 50% (100k), 58.5% (200k)
  2. Rec+Pr+Random - 36% (100k), 50% (200k)
 
I wonder if you know the reason why the performance decline occurred like this. Perhaps we used different pretrained model or hyperparameters, but it will be very helpful if I get your feedback.

**Comment by cailile** [comment]
The result.png is the output of 1 run using the code provided in this repo. The variance of different runs is small (<1%), so such significant drop in your run should be due to other reasons. Are you using the same setting as used in this repo? If you change the batch size or input size due to your GPU memory constraints, or forgot to download the pretrained weights for the backbone model, then the accuracy may drop significantly.

**Comment by cskhy16** [comment]
Thank you for your prompt response!! Except project_dir and number_of_gpu, I use the same hyperparameters used in write_bash_files.py. I wonder if there are problems with the following values: batch_size=4, eval_size=[1025, 2049], and train_crop_size=769.

For a pretrained model, I use "xception_65_imagenet" (447MB) from [deeplab model zoo](https://github.com/tensorflow/models/blob/master/research/deeplab/g3doc/model_zoo.md ). I wonder if I miss anything about the pretrained model.

**Comment by cailile** [comment]
I would suggest to run a fully-supervised baseline (using all the training labels) to see whether you can get an mIoU of ~76%, so that we can tell whether the problem is in the hyperparameters or the AL part.

**Comment by cailile** [comment]
I think the number of GPU can be a problem. I feel that multi-GPU training sometimes can be tricky (e.g., the updating of batch norm parameters). Using a single GPU with smaller crop size (eg. 513) and train longer may help. Good luck!

**Comment by cskhy16** [comment]
Thank you very much for your various advice!!
